src/PowerModels.jl

```diff
@@ -35,6 +35,8 @@ function logger_config!(level)
 end
 
 const _pm_global_keys = Set(["time_series", "per_unit"])
+const _pm_it_name = "ep"
+const _pm_it_sym = :ep
```

Comment:
Maybe this should be `const _pm_it_sym = Symbol(_pm_it_name)`?

Reply:
I am also wondering if `_pm_it_name` (and the symbol) should be an exported parameters `pm_it_name`?  If the multi-infrastructure packages need access these things then probably they should not be marked as internal use only.  Thoughts?

Reply:
That's a good thought. They truly are being used outside the package scope at this point, so let's make these variables public.